dscript/commands/predict.py

```diff
@@ -98,48 +96,43 @@ def main(args):
             ).cpu()
             model.use_cuda = False
     except FileNotFoundError:
-        print(f"Model {modelPath} not found")
-        log(f"Model {modelPath} not found", file=logFile)
+        log(f"Model {modelPath} not found", file=logFile, print_also=True)
         logFile.close()
         sys.exit(1)
 
     # Load Pairs
     try:
+        log(f"Loading pairs from {modelPath}", file=logFile, print_also=True)
         pairs = pd.read_csv(csvPath, sep="\t", header=None)
         all_prots = set(pairs.iloc[:, 0]).union(set(pairs.iloc[:, 1]))
     except FileNotFoundError:
-        print(f"Pairs File {csvPath} not found")
-        log(f"Pairs File {csvPath} not found", file=logFile)
+        log(f"Pairs File {csvPath} not found", file=logFile, print_also=True)
         logFile.close()
         sys.exit(1)
 
     # Load Sequences or Embeddings
     if embPath is None:
         try:
-            names, seqs = parse(open(seqPath, "r"))
+            names, seqs = parse(seqPath, "r")
```

Comment:
does this need to be parse(open(seqPath...)??

Reply:
Updated `parse` command in commit dfbc81829f9e4f56a2be93fcdf8d7530687d1ba9 to use BioPython SeqIO, which can take a file path or an open file object